### DISCUSSION TITLE
Will queueInfo.getTotalMessage appear to be -1?

### DISCUSSION THREAD

**Discussion by Yuan-zs** [rabbitmq/rabbitmq-server#10349] [Questions]
Hi, 
I noticed that the property totalMessage of (com.rabbitmq.http.client.domain)QueueInfo is assigned a value of -1 in the constructor. Are there some rare cases such as server problems or other issues that cause  
`QueueInfo queueInfo = client.getQueue("xx","xx");`
`queueInfo.getTotalMessage() == -1  //true`?
Thank you.

**Comment by michaelklishin** [comment]
If the value was never set, for example, because the metric was not computed at the moment you have requested it, it can be set to something like -1 by the JSON (de)serializer.

RabbitMQ itself may return a value like this for some metrics but IIRC the total counts is not one of them.

**Reply by Yuan-zs** [reply]
Thanks for the reply.
